17.md

```diff
@@ -0,0 +1,194 @@
+NIP-17
+======
+
+Private Direct Messages and Group DMs
+-------------------------------------
+
+`draft` `optional`
+
+This NIP establishes an encrypted direct messaging scheme for 2+ participants with the following privacy and security guarantees: 
+
+1. **Superior Encryption**: Messages are encrypted using [NIP-44](44.md) to each participant's public key individually.
+2. **No Metadata Leak**: Participant identities, each message's real date and time, event kinds, and other event tags are all hidden from the public
+3. **Onion Layers**: Senders and receivers cannot be linked with public information alone ([NIP-59](59.md)).
+4. **Trust-minimized**: Participants cannot expose verifiable details of your message, including the metadata, without reducing the security of their main private key
+5. **No Identifiers**: There is no central queue, channel or otherwise converging identifier to correlate and count all messages in the same group. 
+6. **No Moderation**: There are no group admins: no invitations or bans.
+7. **No Shared Secrets**: There are no chatroom secrets that can leak or be mistakenly shared
```

Comment:
The conversation key is still a long-lived secret between two people that could leak.

Reply:
Nice! I got rid of it for now. I still think something can be said about trust-minimization but frankly, this whole section will be deleted before we merge anyway...